### PR TITLE
Projects: Removed difficulty modifier from Fixed funding model

### PR DESCRIPTION
**Pull request by PeterMPhillips**
Removed difficulty modifiers from the fixed funding model and added remove buttons to both fixed and hourly.

Screenshot from 2019-11-19 09-44-07
Screenshot from 2019-11-19 09-44-20
Screenshot from 2019-11-19 09-50-53

### CODE REVIEW

**Comment by stellarmagnet** [comment]
When it has no multiplier, the label should say just “difficulty”

**Comment by PeterMPhillips** [comment]
New label:
Screenshot from 2019-11-19 12-08-40

File: apps/projects/app/components/Content/Settings.js [hunk 1]
```diff
@@ -502,4 +548,25 @@ const Column = styled.div`
...
+  all: unset;
+  color: ${({ disabled, theme }) => (disabled ? theme.disabled : theme.contentSecondary)};
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
+  display: flex;
+  justify-content: center;
+  :hover {
+    color: ${({ disabled, theme }) => disabled ? theme.disabled : theme.contentBorderActive};
+  }
+  :active {
+    color: ${({ disabled, theme }) => (disabled ? theme.disabled : theme.accent)};
```
Comment (chadoh) [inline apps/projects/app/components/Content/Settings.js]: Let's wrap the whole set of hover & active styles in a conditional. We don't need them if `disabled`. So:

```js
${p => !p.disabled && css`
  :hover { ... }
  :active { ... }
`}

File: apps/projects/app/components/Content/Settings.js [hunk 2]
```diff
@@ -114,37 +116,71 @@ SettingLabel.propTypes = {
...
+                css="font-size: 16px; flex-grow: 1"
+                defaultValue={exp.name}
+                onChange={generateExpLevelHandler(index, 'N')}
+              />
+              <IconContainer
+                theme={theme}
+                disabled={expLevels.length <= 1}
+                style={{ transform: 'scale(.8)' }}
+                onClick={() => onRemoveExpLevel(index)}
+                title="Remove this difficulty multiplier"
```
Comment (chadoh) [inline apps/projects/app/components/Content/Settings.js]: This should say "Remove this difficulty" or maybe "Remove this difficulty setting," but should not say "multiplier."

File: apps/projects/app/components/Content/Settings.js [hunk 3]
```diff
@@ -502,4 +548,25 @@ const Column = styled.div`
...
+    color: ${({ disabled, theme }) => disabled ? theme.disabled : theme.contentBorderActive};
+  }
+  :active {
+    color: ${({ disabled, theme }) => (disabled ? theme.disabled : theme.accent)};
+  }
+  > svg {
+    color: inherit;
+    height: 40px;
+    width: 40px;
+    transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
```
Comment (chadoh) [inline apps/projects/app/components/Content/Settings.js]: Fancy!

File: apps/projects/app/components/Content/Settings.js (hunk 1 above)
Comment (PeterMPhillips) [reply apps/projects/app/components/Content/Settings.js]: Does this work for you?
```
${({ disabled, theme }) => (disabled ? `
      color: ${theme.disabled};
      cursor: not-allowed;
    ` : `
      color: ${theme.contentSecondary};
      cursor: pointer;
      :hover {
        color: ${theme.surfaceOpened};
      }
      :active {
        color: ${theme.accent};
      }
    `)
}
```

**Comment by PeterMPhillips** [comment]
Good catch @chadoh. It turns out the fields values were being set with `defaultValue` instead of `value` so they weren't getting updated on changes

**Comment by javieralaves** [comment]
@PeterMPhillips I'm not able to reproduce these changes [here](https://rinkeby.aragon.org/#/neudune/0xe3fc72a7f3dda98a0a76863c9aad73968c6c0c42/) when switching between Fixed and Hourly, am I doing something wrong?

**Pull request merged**